Rubberduck.RibbonDispatcher/Sample Ribbons/SampleRibbon/src/MyResourceManager.cls

```diff
@@ -0,0 +1,109 @@
+VERSION 1.0 CLASS
+BEGIN
+  MultiUse = -1  'True
+END
+Attribute VB_Name = "MyResourceManager"
+Attribute VB_GlobalNameSpace = False
+Attribute VB_Creatable = False
+Attribute VB_PredeclaredId = False
+Attribute VB_Exposed = False
+Option Explicit
+
+Private Const ModuleName As String = "MyResourceManager."
+
+Private Type TRibbonStrings
+    Strings As Rubberduck_RibbonDispatcher.ControlStrings
+End Type
+Private this As TRibbonStrings
+
+Implements Rubberduck_RibbonDispatcher.IResourceManager
+
+Private Function IResourceManager_GetCurrentUIString(ByVal Name As String) As String
+    On Error GoTo EH
+    IResourceManager_GetCurrentUIString = this.Strings.Item(Name)
+
+XT: Exit Function
+EH: ReraiseError Err, ModuleName & "IResourceManager_GetCurrentUIString"
```

Comment:
Error handler is redundant. Remove.

Reply:
Not until you describe how a VBA stack trace is generated and returned to the user on an error.

Reply:
There is no programmatically accessible stack trace in VBA, and simulating one is an ultimately doomed exercise.